File: 기초 사고력문제/백준_2577_숫자의 갯수/숫자의 갯수.cpp
#include <iostream>

using namespace std;

int main() {
    int arr[10] = {0};
    int a,b,c;
    scanf("%d\n%d\n%d", &a, &b, &c);
    int mul = a*b*c;
    while(mul >= 1){
        arr[mul % 10]++;
        mul /= 10;
    }
    for(int i = 0; i< 10; i++) printf("%d\n", arr[i]);
}

File: 기초 사고력문제/백준_2577_숫자의 갯수/숫자의 갯수.py
a = int(input())
b = int(input())
c = int(input())
mul = a*b*c
arr = [0 for _ in range(10)]

while mul >= 1:
    i = int(mul % 10)
    arr[i] += 1
    mul /= 10

for i in range(len(arr)):
    print(arr[i])
